app/validators/bill-runs/setup/year.validator.js

```diff
@@ -0,0 +1,42 @@
+'use strict'
+
+/**
+ * Validates data submitted for the `/bill-runs/setup/{sessionId}/year` page
+ * @module YearValidator
+ */
+
+const Joi = require('joi')
+
+const VALID_VALUES = [
+  '2024',
```

Comment:
will this require a code change every year to add in new options? should this list be computed?

Reply:
See the PR description! 😁